code/game/objects/items/twohanded.dm

```diff
@@ -661,7 +661,8 @@
 		user.visible_message("<span class='danger'>[user] blasts \the [target] with \the [src]!</span>")
 		playsound(target, 'sound/magic/disintegrate.ogg', 100, 1)
 		W.break_wall()
-		return 1
+		W.ScrapeAway()
```

Comment:
W.dismantle_wall() will cover these two lines

Reply:
Dismantle_wall also plays the welder noise and I think just having disintegrate.ogg fits better